js/DataSourceGlobalFilter.js

```diff
@@ -41,6 +41,10 @@ var DataSourceGlobalFilter = DataSourceIndexed.extend('DataSourceGlobalFilter',
     get: function(filter) {
         return this.filter;
     },
+    
+    sortGroups: function(sorter){
+      this.dataSource.sortGroups(sorter);  
+    },
```

Comment:
why is this here? sounds like it needs to be in base.... but wait didn't you state that filter is closer to origin? is this really being called?